tests/sentry/uptime/endpoints/test_project_uptime_alert_details.py

```diff
@@ -0,0 +1,22 @@
+from sentry.testutils.cases import APITestCase
+
+
+class ProjectUptimeAlertDetailsEndpointTest(APITestCase):
+    endpoint = "sentry-api-0-project-uptime-alert-details"
+
+    def setUp(self):
+        super().setUp()
+        self.login_as(user=self.user)
+
+    def test_simple(self):
+        uptime_subscription = self.create_project_uptime_subscription()
+
+        resp = self.get_success_response(
+            self.organization.slug, uptime_subscription.project.slug, uptime_subscription.id
+        )
+        assert resp.data["id"] == str(uptime_subscription.id)
+        assert resp.data["name"] == uptime_subscription.name
```

Comment:
Usually for apis I just do `resp.data === serialize(uptime_subscription, self.user)`